### PR TITLE
export ValidationError and MissingRefError

### PR DESCRIPTION
**Pull request by Ginden**
**What issue does this pull request resolve?**

#1832

**What changes did you make?**

Exported 2 Error subclasses.

**Is there anything that requires more attention while reviewing?**

### CODE REVIEW

**Comment by epoberezkin** [comment]
Never saw this export format before :)

It should be added in other ajv classes too - core.ts, jtd.ts, 2019.ts and 2020.ts

**Comment by epoberezkin** [comment]
Probably no need to export from core.ts.

File: lib/ajv.ts [hunk 1]
```diff
@@ -65,3 +65,5 @@ export {DefinedError} from "./vocabularies/errors"
 export {JSONType} from "./compile/rules"
 export {JSONSchemaType} from "./types/json-schema"
 export {_, str, stringify, nil, Name, Code, CodeGen, CodeGenOptions} from "./compile/codegen"
+export {default as ValidationError} from './runtime/validation_error'
```
Comment (epoberezkin) [inline lib/ajv.ts]:
```suggestion
export {default as ValidationError} from "./runtime/validation_error"
```

**Comment by dannyb648** [comment]
What is required to get this over the line? Im hitting alot of issues trying to upgrade a codebase. Happy to get involved if possible

**Comment by epoberezkin** [comment]
@dannyb648: It should be added in other ajv classes too - jtd.ts, 2019.ts and 2020.ts

**Comment by dannyb648** [comment]
> @dannyb648: It should be added in other ajv classes too - jtd.ts, 2019.ts and 2020.ts

I've added these changes and the original ones to PR #2020 - Any chance of a review? Additionally looks like something is broken with the coverage plugin on all the PRs atm?

**Pull request closed without merging**